addons/hr_payroll/models/hr_salary_rule.py

```diff
@@ -29,6 +33,8 @@ def _get_parent(self):
     parent_id = fields.Many2one('hr.payroll.structure', string='Parent', default=_get_parent)
     children_ids = fields.One2many('hr.payroll.structure', 'parent_id', string='Children', copy=True)
     rule_ids = fields.Many2many('hr.salary.rule', 'hr_structure_salary_rule_rel', 'struct_id', 'rule_id', string='Salary Rules')
+    report_id = fields.Many2one('ir.actions.report',
+        string="Report", required=True, domain="[('model','=','hr.payslip'),('report_type','=','qweb-pdf')]", default=_get_report_payslip)
```

Comment:
Why `('report_type','=','qweb-pdf')` ? You could want to print an HTML report  no ?